paper/basic_training.tex

```diff
@@ -533,7 +534,10 @@ \subsection{Main steps of a molecular dynamics simulation}
 It is highly recommended that best practices documents specific to this issue and to the type of system of interest be consulted.
 If such documents do not exist, it may be that a freely-available tool for constructing such a system does in fact exist.
 Examples include tools for constructing specific crystal structures, proteins, and lipid membranes.
-\todo[inline, color={green!20}]{JIM: Include some tools to cite here}
+\todo[inline, color={green!20}]{JIM: Include some tools to cite here.
+EB: Many tools are listed at
+\url{https://lammps.sandia.gov/prepost.html#builders}. My favorites include
```

Comment:
Feel free to just go ahead and add. I'm a fan of packmol as well. Haven't used the others.

Reply:
Fixed.